docs/getting-started/user-guides/testnet-voting.md

```diff
@@ -0,0 +1,183 @@
+
+> Step 3: Edit Config Files to Launch Testnet
+
+To run `dcrd`, `dcrwallet`, and `dcrctl` on the testnet, simply add `testnet=true` or `testnet=1` to all three configuration files. If you are using one of the sample config files, you can simply find the line that reads `;testnet=1` (the first setting within the Network Settings section) and delete the semi-colon. 
+
+This must be done for all three configuration files. 
+
+> Step 4: Create a New Testnet Wallet
+
+If you've never run a Testnet wallet before, you will need to create a new one. With your `dcrwallet.conf` setup for testnet **(see Step 3)**, run `dcrwallet` with the `--create` flag. 
+
+For those unfamiliar with how to do that, follow the operating system specific guide below:
+
+**Windows**: <br />
+    1. Using Command Prompt or File Explorer, navigate to the directory of your `dcrwallet` executable <br />
+    2. If using File Explorer, select "Open command prompt" option from the "File" dropdown menu <br />
+    3. Enter the `dcrwallet.exe --create` command
+
+**macOS**: <br />
+    1. Using Terminal or Finder, navigate to the directory of your `dcrwallet` executable <br />
+    2. If using Finder, you can open a new Terminal at a folder location by right-clicking a folder and selecting Services > New Terminal at Window from the dropdown menu<br />
+    3. Enter the `./dcrwallet --create` command
+
+**Linux**: <br />
+    1. Using anything you like, navigate to the directory of your `dcrwallet` executable <br />
+    2. Enter the `./dcrwallet --create` command
+
+This will run you through the usual prompts of creating a new wallet. Follow the directions on the screen. You will be required to create a private passphrase (you will use this later to unlock your wallet while creating transactions). The additional layer of encryption is completely optional. Your seed can be used to restore your wallet on any computer using dcrwallet. Record your seed, put it in a safe place, and never share it with anyone. The wallet will close once everything is finished.
+
+> Step 5: Start dcrd on the Testnet
+
+Run your `dcrd` executable with the `testnet=1` or `testnet=true` option within the config file to start your node on the testnet. Your node will begin to sync with the rest of the network. Syncing will take awhile.
+
+> Step 6: Start dcrwallet on the Testnet
+
+Run your `dcrwallet` executable with the `testnet=1` or `testnet=true`option within the config file to start your wallet on the testnet. Your wallet will connect to your node and begin to sync your addresses. Syncing may take awhile.
+
+> Step 7: Register at Stakepool Website
+
+While waiting for your node/wallet to sync, visit [https://teststakepool.decred.org](https://teststakepool.decred.org) and register for a new account. Continue to step 8.
+
+> Step 8: Wait for Testnet Node/Wallet to Sync
+
+Take a break, this may take awhile. Your CPU will be taxed heavily during this syncing.
```

Comment:
More disk intensive than CPU really.  And I've done this on a raspberry pi multiple times, so not that taxing.  Don't want to scare people away thinking this requires some crazy hardware.  Maybe soften that language or remove and just leave it as 'may take a while'.